features/FEATURE_UVISOR/source/rtx/box_init.c

```diff
@@ -29,7 +29,7 @@
 extern void SVC_Handler(void);
 extern void PendSV_Handler(void);
 extern void SysTick_Handler(void);
-extern uint32_t svcRtxKernelLock(void);
```

Comment:
That's going to be overwritten by next uvisor drop.

Reply:
It was merged by uvisor, the new update contains this, its ready for merge: https://github.com/ARMmbed/mbed-os/pull/4907/files .  This will be rebased once it gets in

Reply:
This PR shouldn't do the update, but should use the uVisor v0.30.0 changes.

Does this PR need to raise the priority of PR #4907? We thought the v0.30.0 PR was optional for 5.6, but maybe because this PR depends on it, the uVisor PR must go into 5.6.

Reply:
Thanks @Patater. If both gets in, it would be nice. This is nice to have.
We will wait for uvisor to get in with the change, and rebase this one afterwards.